src/app-shell.ts

```diff
@@ -19,24 +23,25 @@ export class AppShell extends LitElement
     {
         return html`${this.renderView()}`;
     }
+
     @state()
     private currentRoute = 'login';
 
-    private updateRoute(event: CustomEvent): void
+    override stateChanged(state: any):void
```

Comment:
I think (not sure) this Method is or should be protected, not public.
Also please use a typed state.